.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "restructuredtext.confPath": "${workspaceFolder}/docs"
+}
```

Comment:
Looks like this was included by accident. Feel free to change `.gitignore` to add this path to the list of ignored files.

Reply:
Will do

Reply:
Could you delete this file from the PR too, please?